Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml

```
@@ -48,29 +56,39 @@ configuration:
   - dlp
   required: false
   type: 16
+  section: Collect
+  advanced: true
```

```suggestion
  section: Collect
```
I think it will not fetch anything if this is empty